README.md

```diff
@@ -98,3 +98,35 @@ remove the .spec dictionary from the yaml)
 
 NOTE: Deploying from the catalog will only work in the opendatahub namespace
 since the operator will already be deployed
+
+## Sample Demo Deployments
+
+### OptaPlanner Demo
+To demonstrate OptaPlanner with the dashboard and admin screens with a smaller resource requirement, we can skip several portions of the demo.  Using the environment file similar to the following:
+```.env
+# oc_login
+OC=~/bin/oc
+OC_URL=https://openshift-host:8443
+OC_USER=admin
+OC_PASSWORD=admin
+
+# Datagrid
+DATAGRID_INSTANCES=1
+DATAGRID_CPU=1000m
+DATAGRID_MEMORY=2048Mi
+
+# OptaPlanner
+OPTAPLANNER_REQUEST_CPU=1000m
+OPTAPLANNER_REQUEST_MEMORY=2000Mi
```

Comment:
During the demo with 1000 users OptaPlanner's memory consumption was <500m so it's better to keep the request here at 1000Mi

Reply:
done